test/ModuleInterface/stdlib.swift

```diff
@@ -1,3 +1,4 @@
+// RUN: %target-swift-frontend -typecheck -emit-interface-path - -parse-stdlib %s | %FileCheck %s
 // RUN: %target-swift-frontend -emit-interface-path - -emit-module -o /dev/null -parse-stdlib %s | %FileCheck %s
```

Comment:
I think this line can be removed

Reply:
Ah, I wanted to have at least one test that did things both ways; a -parse-stdlib test seemed like a good example.

Reply:
Sounds good, then!